heartbeat/server.js

```diff
@@ -175,24 +176,34 @@ function checkHeartbeat() {
             let flicDelayMillis = currentTime.diff(flicLastSeenTime)
             let heartbeatLastSeenTime = moment(doc.heartbeat_last_seen_time)
             let heartbeatDelayMillis = currentTime.diff(heartbeatLastSeenTime)
+            let pingLastSeenTime = moment(doc.flic_last_ping_time)
```

Comment:
I noticed that `flic_last_ping_time` should be `ping_last_seen_time` to match the other two, however it looks like it's already being used in the "DB".

@mariocimet does the Heartbeat use Postgres in the combined Buttons server?

Reply:
it does yes, these NeDB calls all get switched

Reply:
I agree, `ping_last_seen_time` makes more sense.

Reply:
:+1: So this will be handled in PR #53